both/routes.js

```diff
@@ -25,27 +25,26 @@ if(Meteor.isClient) {
     return Router.current().route.getName() == routeName;
   });
 
-  Template.registerHelper("refreshTitle", function() {
-    var title = "live.cubing.net";
+  Template.registerHelper("documentTitle", function() {
```

Comment:
I agree that *documentTitle* is too bland. But isn't every reactive function "refreshing" something?

Is *setDocumentTitle* any better? *setHtmlTitle*?